build-system/status-page/comment.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview
+ * Add progress comment for Stable and LTS cherry-picks
+ */
+
+const fetch = require('node-fetch');
+const [number, body, user] = process.argv.slice(2);
+const {getChannels, steps} = require('./common');
+const {log} = require('../common/logging');
+
+const commentTemplate = (channels) => `
+  #### 🌸 Cherry-Pick Progress 🌸 
+  Hi @${user}, thanks for filing this cherry-pick request!
+  Seeing that this affects ${channels.join(
+    ' and '
+  )}, [status.amp.dev](https://status.amp.dev) will be updated with progress of the fix.
+  Please update this tracker as each step is completed.
+  - [ ] ${steps[0].text}
+  - [ ] ${steps[1].text}
+  - [ ] ${steps[2].text}
+  - [ ] ${steps[3].text}
```

Comment:
steps.map(...) + join on '\n' plz?

Reply:
Like this?
```
${steps.map((step) => {
  return `- [ ] ${step.text}`
}).join('\n')}
```
The original way is a lot more readable though

Reply:
If the idea is to avoid enumerating 0 / 1 / 2 / 3, what if each step used a named index that's referenced here?

Reply:
`steps` benefits from being an array because I can iterate, a good example is shown in Daniel's comment below.

This feels like a nit... does anyone have a strong preference?

Reply:
No strong preference here, okay to leave as is 👍

Reply:
Ok fair! PTAL